handlers/error.go

```diff
@@ -52,7 +52,7 @@ const (
 	ErrRetrieveUserTokenCode    = "2044"
 	ErrFailToSaveCode           = "2045"
 	ErrFailToDeleteCode         = "2046"
-	ErrFailToLoadExtensionsCode = "2047"
+	ErrFailToLoadExtensionsCode = "2048"
```

Comment:
Hey @Shreyas220 let `ErrFailToLoadExtensionsCode = "2047"` only change the `https://github.com/meshery/meshery/blob/master/component_info.json ` file `next_error_code` to "2048".

Reply:
oh, so sorry ......didn't read the comment properly  :p